apps/portals-base/src/components/Versions.tsx

```diff
@@ -59,13 +59,13 @@ const Versions: React.FunctionComponent = () => {
 
   return (
     <>
-      {deployDate && buildDate && (
+      {(deployDate || buildDate) && (
         <Tooltip
           title={
             <div style={{ textAlign: 'center' }}>
-              Built: {buildDate}
+              {buildDate && <>Built: {buildDate}</>}
               <br />
-              Deployed: {deployDate}
+              {deployDate && <>Deployed: {deployDate}</>}
```

Comment:
Update so both are not required to display (so I could manually verify that displaying `buildDate` worked without creating a deploy-date.txt)